pages/workflow/review.mdx

```diff
@@ -0,0 +1,147 @@
+---
+layout: 'guide'
+title: '2. Review suggestions in Avo'
+abstract: ''
+---
+
+import Link from '../../components/Link';
+import styles from '../index.module.scss';
+import PageLink, { CallToAction } from '../../components/PageLink';
+import AnalyticsManagerIcon from '../../images/svg/analyticsmanager.svg';
+import DevIcon from '../../images/svg/dev.svg';
+
+import { FunctionComponent } from 'react';
+import Zoom from 'react-medium-image-zoom';
+
+
+
+## Ask for a review
+
+_Role: Product, Data, Engineering. See more below _
+
+
+Once you have drafted changes to your tracking plan we recommend asking your teammates to review your suggested changes.
+
+**Who should be asked for a review?**
+We typically see three types of reviewers:
+- **Product team:** Someone who is responsible for the related feature or user flow that verifies that the metrics mapped out on the branch suffice for their data needs
+- **Data team:** Someone who validates that the changes made are according to the data standard of your company
+- **Engineering team:** The folks who will implement the suggested changes. If the changes made affect multiple sources (e.g. iOS, Android, Web, backend) we recommend to get reviews from the teams working on each source
+
+**How to ask for a review?**
+The first step is to mark the branch as “Ready for review”. This is done at the bottom of the review screen as seen on the screenshot below.
+
+<Zoom>
+<center>
+    <img
+        width="60%"
+        src={require('../../images/workflow/2.review/ready-for-review.png')}
+        alt="even definition"
+    />
+</center>
+</Zoom>
+
+_Example: Changes ready for review_ 
+
+> Note that detailed branch statuses like “Draft”, “Ready for Review”, “Request changes” and “Approved” are only available on our paid plans. If you’re currently on the Free plan you can start a free trial today to try it out.
+
+There are couple ways to share a review request:
+- Add the reviewers as [branch collaborators](https://www.avo.app/blog/introducing-branch-collaborators): Go to the branch review screen of your branch and add your reviewers as collaborators. A collaborator will receive an email when added to your branch, when discussions are started, when the branch status is updated and when the branch is merged or closed.
+
+<video
+  controls
+  loop
+  muted
+  width="100%"
+  autoPlay
+  src={require('../../images/workflow/2.review/add-collaborators.mp4')}
+  alt="Add Collaborators"
+/>
```

Comment:
Can we make this video have the same indent as the bullet it's attached to? Maybe just removing the newline between them fixes it?
<img width="924" alt="Screenshot 2022-07-21 at 22 18 02" src="https://user-images.githubusercontent.com/3513968/180324483-4b078358-9b99-4f6f-a982-f76923136608.png">